.github/workflows/autoremove_trivyignore.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: Auto-remove ignored CVEs
+on:
+  workflow_call:
+
+jobs:
+  update-lib:
+    permissions:
+      id-token: write  # Enable OIDC
+      pull-requests: write
+      contents: write
+    name: Autoremove ignored CVEs
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+        with:
+          fetch-depth: 0
+      - name: Check trivyignore
+        run: |
+          curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin v0.18.3
+          if [ -f ".trivyignore" ]
+          then
+            output=$(trivy fs . -q -f json --ignorefile "" | jq -r '.Results[].Vulnerabilities[].VulnerabilityID' 2>/dev/null || echo "No vulnerabilities found")
```

Comment:
The [trivyignore](https://aquasecurity.github.io/trivy/v0.46/docs/configuration/filtering/#by-finding-ids) might have more than CVEs. Would be possible to compare the output running without trivyignore and with trivyignore and then create a PR to check the difference?

Reply:
I'm resolving this as we're going to annotate the trivyignore against CVEs that are outdated, so comparing output of CVEs (without trivyignore) against these present in the trivyignore and annotate the file. No PR will be created, we will be using the approach that we have with hadolint (annotations). Outdated CVEs are a chore, so that's a good solution imo.